messaging/src/main/java/org/axonframework/common/infra/DescribableComponent.java

```diff
@@ -18,7 +18,22 @@
 
 import jakarta.annotation.Nonnull;
 
+/**
+ * Contract allowing components {@link #describeTo(ComponentDescriptor) to describe} themselves with a given
+ * {@link ComponentDescriptor}.
```

Comment:
Nit: I think it would be good to add a sentence that the given `ComponentDescriptor` is key in deciding the formatting of the outcome.

Reply:
Added, please check it out :)

Reply:
Looks great now, thanks dude!